hdf5_record_server:
  hdf5_save_interval: 500
